README.md

```diff
@@ -39,6 +39,8 @@ Additionally, an [example configuration](https://github.com/prometheus/blackbox_
 HTTP, HTTPS (via the `http` prober), DNS, TCP socket and ICMP (see permissions section) are currently supported.
 Additional modules can be defined to meet your needs.
 
+The timeout of each probe may be configured using `timeout` in the Blackbox exporter config file or preferably by setting the `scrape_timeout` in the [Prometheus config](https://prometheus.io/docs/operating/configuration/#configuration-file) itself.
```

Comment:
This doesn't quite capture the logic. 

The timeout of each probe is automatically determined from the `scrape_timeout` in the Prometheus config, slightly reduced to allow for network delays. This can be further limited by the `timeout` in the Blackbox exporter config file. If neither is specified, it defaults to 10 seconds.